addons/account/wizard/account_merge_wizard.py

```diff
@@ -0,0 +1,310 @@
+from odoo import _, api, fields, models, Command
+from odoo.exceptions import UserError
+from odoo.tools import SQL
+
+
+class AccountMergeWizard(models.TransientModel):
+    _name = 'account.merge.wizard'
+    _description = "Account merge wizard"
+
+    account_ids = fields.Many2many('account.account')
+    is_group_by_name = fields.Boolean(
+        string="Group by name?",
+        default=True,
```

Comment:
Why the default to True?
I think it is going to be used more by trying to merge the accounts 2 by 2 with manual groups, and often with different names (different CoA). So the users will often have to untick the box.

Whereas the case where we need to tick should happen only once, so it's not an issue to click it only one time.

Reply:
Discussed with Benj, we'll indeed default it to False